applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.hpp

```diff
@@ -414,36 +415,34 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ShellCrossSection : public Fl
 
     private:
 
-        double mThickness;
-        double mLocation;
+        int mPlyIndex;
         double mOrientationAngle;
         IntegrationPointCollection mIntegrationPoints;
         Properties::Pointer mpProperties;
 
     public:
 
         Ply()
-            : mThickness(0.0)
-            , mLocation(0.0)
-            , mOrientationAngle(0.0)
+            : mPlyIndex(0)
             , mIntegrationPoints()
             , mpProperties(Properties::Pointer())
         {}
 
-        Ply(double thickness, double location, double orientationAngle, int numPoints, const Properties::Pointer & pProperties)
-            : mThickness(thickness)
-            , mLocation(location)
+        Ply(const int PlyIndex, int NumIntegrationPoints, const Properties::Pointer & pProperties)

```

Comment:
Is it intentional to not initialized `mOrientationAngle`. Did you mean to completed remove this variable from the class?

Reply:
Yep I forgot to completely remove it